### PR TITLE
feat: SendAI integration - Solana Agent Kit

### PR DESCRIPTION
**Pull request by 0xtarunkm**
- [x] Create NFT Collection
- [x] Swap Tokens
- [x] Lend Tokens
- [x] Stake SOL
- [x] Fetch Token Price
- [x] SPL Token Airdrop via ZK Compression

### CODE REVIEW

File: docs/sample-requests/bot.start.json [hunk 1]
```diff
@@ -16,6 +16,12 @@
         "description": "",
         "prompt": "You are nerosdk, check sentiment of the given message and roast elon musk. Keep replies to 2 lines. Don’t use hashtags"
       }
-    ]
+    ],
+    "createaNftCollection": true,
```
Comment (CodingTux) [inline docs/sample-requests/bot.start.json:20]: let's move sendai config inside "sendai" blocks? wdyt

File: src/api/bot.ts [hunk 2]
```diff
@@ -71,6 +36,49 @@ router.post<{}>("/start", async (req, res, next) => {
       mentioned_handle: details?.metadata?.twitter_handle || "nerobossai",
       prompt: details.replies_prompt,
     });
+
+    if (details.nftCollection) {
+      nftcreationqueue.push({
+        mentioned_handle: details?.metadata?.twitter_handle || "nerobossai",
```
Comment (CodingTux) [inline src/api/bot.ts:42]: let's create constant on top or create some default config for constants. eg: nerobossai here

**Comment by 0xtarunkm** [comment]
https://github.com/user-attachments/assets/4ab1f005-a44a-4e63-8d71-98b62007620f

@CodingTux

File: src/api/bot.ts [hunk 3]
```diff
@@ -64,13 +30,64 @@ router.post<{}>("/start", async (req, res, next) => {
...
+
+    // Create Token
     tokencreationqueue.push({
-      mentioned_handle: details?.metadata?.twitter_handle || "nerobossai",
+      mentioned_handle: details?.metadata?.twitter_handle || DEFAULT_X_HANDLE,
       prompt: details.replies_prompt,
     });
+
+    // create NFT collection using metaplex code
+    if (details.nftCollection) {
```
Comment (CodingTux) [inline src/api/bot.ts:44]: it should be ig details.sendai.createaNftCollection?

Comment (CodingTux) [reply src/api/bot.ts:44]: same with other if conditions

File: docs/sample-requests/bot.start.json [hunk 4]
```diff
@@ -16,6 +19,14 @@
         "description": "",
         "prompt": "You are nerosdk, check sentiment of the given message and roast elon musk. Keep replies to 2 lines. Don’t use hashtags"
       }
-    ]
+    ],
+    "sendai": {
+      "createaNftCollection": true,
```
Comment (CodingTux) [inline docs/sample-requests/bot.start.json:24]: also typo here

**Comment by CodingTux** [comment]
LGTM ✅

**Review by CodingTux — APPROVED** [review]

**Pull request merged**